Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -393,6 +393,24 @@ void ParseRadicalLine(RWMol *mol, const std::string &text, bool firstCall,
   }
 }
 
+void ParsePXALine(RWMol *mol, const std::string &text, unsigned int line) {
+  PRECONDITION(mol, "bad mol");
+  PRECONDITION(text.substr(0, 6) == "M  PXA", "bad PXA line");
+  unsigned int pos = 7;
+  try {
+    unsigned int atIdx =
+        FileParserUtils::stripSpacesAndCast<unsigned int>(text.substr(pos, 3));
+    pos += 3;
+    mol->getAtomWithIdx(atIdx - 1)->setProp(
+        "_MolFile_PXA", text.substr(pos, text.length() - pos));
+  } catch (boost::bad_lexical_cast &) {
+    std::ostringstream errout;
+    errout << "Cannot convert " << text.substr(pos, 3) << " to int on line "
```

Comment:
Perhaps add

```
"Cannot convert '" << ... << "' to int"
```

I sometimes get annoyed when I don't know when the actual invalid text starts and ends.